.github/validation_samples/kaon_enhanced/config.py

```diff
@@ -0,0 +1,162 @@
+import os
+import sys
+
+from LDMX.Framework import ldmxcfg
+
+thisPassName = 'test'
+p=ldmxcfg.Process(thisPassName)
+
+p.termLogLevel = 0
+# This need to be set but has no meaning given p.totalEvents
+p.maxEvents = 1
+p.totalEvents = int(0.5*int(os.environ['LDMX_NUM_EVENTS']))
+p.run = int(os.environ['LDMX_RUN_NUMBER'])
+nElectrons = 1 
+beamEnergy = 8;  #in GeV      
+
+from LDMX.SimCore import generators as gen
+from LDMX.SimCore import bias_operators
+from LDMX.SimCore import kaon_physics
+from LDMX.SimCore import photonuclear_models as pn
+from LDMX.Biasing import ecal
+from LDMX.Biasing import filters
+from LDMX.Biasing import particle_filter
+from LDMX.Biasing import util
+from LDMX.Biasing import include as includeBiasing
+
+if beamEnergy == 4 :
+     detector = 'ldmx-det-v14'
+     generator=gen.single_4gev_e_upstream_tagger()
+     bias_factor = 450.
+     bias_treshold = 2500.
+else :
+     detector = 'ldmx-det-v14-8gev'
+     generator=gen.single_8gev_e_upstream_tagger()
+     bias_factor = 550.
+     bias_treshold = 5000.
```

Comment:
The thresholds in the filters below are hard-coded to the 8GeV values so I'm removing the 4GeV branch here to avoid hinting to users that they could just change the `beamEnergy` above and it would work.

```suggestion
detector = 'ldmx-det-v14-8gev'
generator=gen.single_8gev_e_upstream_tagger()
bias_factor = 550.
bias_treshold = 5000.
```

Reply:
done in the next push